rtc_tools/rtc_event_log_visualizer/analyzer.cc

```diff
@@ -1199,7 +1199,7 @@ void EventLogAnalyzer::CreateSendSideBweSimulationGraph(Plot* plot) {
   auto factory = GoogCcNetworkControllerFactory();
   TimeDelta process_interval = factory.GetProcessInterval();
   // TODO(holmer): Log the call config and use that here instead.
-  static const uint32_t kDefaultStartBitrateBps = 300000;
+  static const uint32_t kDefaultStartBitrateBps = 100000;
```

Comment:
Same in this file: can we reference one place in api/transport/bitrate_settings.h?